<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <node
        pkg="cuboid_detection"
        type="surface_normal_estimation"
        name="surface_normal_estimation"
        output="screen">

        <!-- Set topics and params -->
        <rosparam>
            <!-- invert (default: true): True > Box; False > Ground Plane -->
            invert: true
            <!-- voxel_size (default: 0.01): Lower the value, denser the point cloud, more CPU -->
            voxel_size: 0.005
            <!-- distance_threshold (default: 0.01): Lower the value, more of box + table noise -->
            distance_threshold: 0.015
            <!-- Topics -->
            input: "/ground_plane_segmentation/points"
            output: "/surface_segmentation/points"
            plane_coefficients: "/ground_plane_segmentation/coefficients"
        </rosparam>
    </node>

</launch>
